src/lib/snakeCase.js

```diff
@@ -0,0 +1,37 @@
+const snakeCase = (startingString) => {
+
+  var array = startingString.split('');
+  var newArray = [];
```

Comment:
I've seen the use of the `newArray` identifier frequently throughout this code - consider naming the identifier with an intention revealing name that highlights the purpose of the variable, instead of the more generic `newArray`.